hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -828,4 +829,17 @@ protected Option<byte[]> readDataFromPath(Path detailPath) {
   public HoodieActiveTimeline reload() {
     return new HoodieActiveTimeline(metaClient);
   }
+
+  public void copyInstant(HoodieInstant instant, Path dstDir) {
+    Path srcPath = new Path(metaClient.getMetaPath(), instant.getFileName());
+    Path dstPath = new Path(dstDir, instant.getFileName());
+    try {
+      FileSystem srcFs = srcPath.getFileSystem(metaClient.getHadoopConf());
+      FileSystem dstFs = dstPath.getFileSystem(metaClient.getHadoopConf());
+      dstFs.mkdirs(dstDir);
+      FileUtil.copy(srcFs, srcPath, dstFs, dstPath, false, srcFs.getConf());
```

Comment:
lets make 5th arg as true. on crash scenarios, if we retry to back up again, we should not fail.

Reply:
5th arg is deleteSource. Changed the API to use a different version of copy which accepts the 6th argument as overwrite flag which is set to true.